storage/backup/backup-cli/src/metadata/mod.rs

```diff
@@ -155,6 +170,9 @@ impl Metadata {
                 format!("transaction_{}-{}.meta", t.first_version, t.last_version)
             },
             Metadata::Identity(_) => "identity.meta".into(),
+            Self::CompactionTimestamps(e) => {
+                Self::get_compaction_timestamps_file_name(e.file_compacted_at)
```

Comment:
should do `format!("..", e.file_compacted_at)` and remove `get_compaction_timestamps_file_name`

Reply:
get_compaction_timestamps_file_name is also used in coordinators/backup.rs. we may want to have the name formating in one place instead of duplicating in multiple files?